pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = finalProject_g4.settings
+python_files = tests.py test_*.py *_tests.py
```

Comment:
Please add a new line at the end of the file

Reply:
@avi-biton Done.

Reply:
LGTM